WORKSPACE

```diff
@@ -44,30 +44,42 @@ load("@aspect_rules_js//js:npm_import.bzl", "npm_import")
 # Just a demonstration of the syntax de-sugaring.
 npm_import(
     name = "example_npm_deps__acorn-8.4.0",
-    integrity = "sha512-ULr0LDaEqQrMFGyQ3bhJkLsbtrQ8QibAseGZeaSUiT/6zb9IvIkomWHJIvgvwad+hinRAgsI51JcWk2yvwyL+w==",
     package_name = "acorn",
-    package_version = "8.4.0",
+    integrity = "sha512-ULr0LDaEqQrMFGyQ3bhJkLsbtrQ8QibAseGZeaSUiT/6zb9IvIkomWHJIvgvwad+hinRAgsI51JcWk2yvwyL+w==",
     link_package_guard = "example",
+    package_version = "8.4.0",
 )
 
 load("@aspect_rules_js//js:translate_pnpm_lock.bzl", "translate_pnpm_lock")
 
+translate_pnpm_lock(
+    name = "aspect_rules_js_lifecycle_npm_deps",
+    pnpm_lock = "//js/private/lifecycle:pnpm-lock.json",
+    # Avoid a circular dependency on the lifecycle hooks package,
+    # which is used when skip_lifecycle_hooks = False
+    skip_lifecycle_hooks = True,
```

Comment:
Usually Booleans should have a true value in the positive case. So it should be enable_...=False

Reply:
Nice, that makes the logic look cleaner as well.